integration_test/chart_values/backend.yaml

```diff
@@ -0,0 +1,2002 @@
+          name: participant
+
+    autoMigrate: false
+
+  resources: {}
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+
+  nodeSelector: {}
+
+  extraEnv: []
+
+  extraVolumes: []
+  # - name: my-volume
+  #   secret:
+  #     secretName: my-secret
+
+  extraVolumeMounts: []
+  # - name: my-volume
+  #   mountPath: /etc/secrets/my-secret
+  #   readOnly: true
+
+  # -- Configuration for tracing providers. Only datadog is currently supported through this block.
+  # If you need to use a different tracing provider, please manually set the configuration values
+  # via "keto.config" or via "extraEnv".
+  tracing:
+    datadog:
+      enabled: false
+
+      # Sets the datadog DD_ENV environment variable. This value indicates the environment where keto is running.
+      # Default value: "none".
+      # env: production
+
+      # Sets the datadog DD_VERSION environment variable. This value indicates the version that keto is running.
+      # Default value: .Chart.AppVersion (i.e. the tag used for the docker image).
+      # version: X.Y.Z
+
+      # Sets the datadog DD_SERVICE environment variable. This value indicates the name of the service running.
+      # Default value: "ory/keto".
+      # service: ory/keto
+
+      # Sets the datadog DD_AGENT_HOST environment variable. This value indicates the host address of the datadog agent.
+      # If set to true, this configuration will automatically set DD_AGENT_HOST to the field "status.hostIP" of the pod.
+      # Default value: false.
+      # useHostIP: true
+
+  tolerations: []
+
+  affinity: {}
+
+  # -- Configure the probes for when the deployment is considered ready and ongoing health check
+  deployment:
+    livenessProbe:
+      initialDelaySeconds: 30
+      periodSeconds: 10
+      failureThreshold: 5
+    readinessProbe:
+      initialDelaySeconds: 30
+      periodSeconds: 10
+      failureThreshold: 5
+
+  # -- Watcher sidecar configuration
+  watcher:
+    enabled: false
+    image: oryd/k8s-toolbox:0.0.2
+    mountFile: ''
+    # mountFile: /etc/secrets/my-secret/foo
+
+  # -- PodDistributionBudget configuration
+  pdb:
+    enabled: false
+    spec:
+      minAvailable: 1
+
+oathkeeper:
+  enabled: false
+
+kratos:
+  enabled: false
+
+kratos-db:
+  enabled: false
+
+################################################################
+## MySQL DB for Keto
+keto-db:
+  enabled: true
```

Comment:
If you want to reduce some workload, you can disable keto-db and change the keto image to sqlite version.

Reply:
The CI manages for now so going to leave as is.